v1/config/config.go

```diff
@@ -95,7 +98,8 @@ type DynamoDBConfig struct {
 
 // SQSConfig wraps SQS related configuration
 type SQSConfig struct {
-	Client          *sqs.SQS
+	ClientV1        *sqs.SQS
```

Comment:
can we please add some comments here describing whats the difference between v1 and v2 ( usage of AWS SDK v1 vs v2, with IMDS support) . Also for backward compatibility might be better to keep original `Client` as-is?

Reply:
@bharat-p updated, please check